agent/engine/engine_unix_integ_test.go

```diff
@@ -1331,3 +1369,41 @@ func TestTaskLevelVolume(t *testing.T) {
 	client := taskEngine.(*DockerTaskEngine).client
 	client.RemoveVolume(context.TODO(), "TestTaskLevelVolume", 5*time.Second)
 }
+
+func TestGPUAssociationTask(t *testing.T) {
+	if os.Getenv("ECS_ENABLE_GPU_SUPPORT") == "false" {
```

Comment:
i think we should probably skip the task if ECS_ENABLE_GPU_SUPPORT is not explicitly set to true